src/cli/src/install/mod.rs

```diff
@@ -13,15 +13,20 @@ pub(crate) fn fluvio_extensions_dir() -> Result<PathBuf, CliError> {
     if let Ok(dir) = std::env::var("FLUVIO_DIR") {
         // Assume this is like `~/.fluvio
         let path = PathBuf::from(dir).join("extensions");
-        if path.exists() {
-            return Ok(path);
+        if !path.exists() {
+            std::fs::create_dir(&path)?;
```

Comment:
actually,  shouldn't not create extension directory if it doesn't exists for purpose of query extensions.   It should be only done when extension is installed

Reply:
I'm fine with that. I'm just not sure what's more graceful than saying
```
Error:
    0: Fluvio extensions directory not found
```

If you're running an extension and there's no extensions directory, the user should know they've gotta reinstall or something. @nicholastmosher What do you think?

Reply:
Hmm.  I think it is ok to create directory if it doesn't exists.